.pipelines/js-ci.yml

```diff
@@ -41,28 +40,34 @@ parameters:
   - unit-tests
 
 steps:
-- task: NodeTool@0
-  name: NodeTool1
-  displayName: Use Node 14.x
+- task: UseNode@1
+  displayName: Use Node 20.x
   inputs:
-    versionSpec: 14.x
+    version: '20.x'
 
-- bash: |
-   npm install -g npm@8.11.0
-  workingDirectory: source/nodejs
-  displayName: 'Bash - npm install -g npm 8.11.0'
-  
-- bash: |
-   npm ci 
-   npx lerna bootstrap --ci
-   npx lerna run release
-  workingDirectory: source/nodejs
-  displayName: 'Bash - lerna bootstrap'
+- task: npmAuthenticate@0
+  displayName: 'npm authenticate'
+  inputs:
+    workingFile: 'source/nodejs/.npmrc'
 
 - bash: |
-   npx lerna run test
+   npm ci --maxsockets=1
```

Comment:
nit: wonder if it's too restrictive.

Reply:
this is a workaround for a bug in npm that affects `npm ci` on Windows machines (like our CI box). I wasn't able to repro locally...